provider/marathon/marathon_test.go

```diff
@@ -257,6 +257,124 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "multiple ports",
```

Comment:
I added them here because they are more then just the port, though the different ports is the most obvious feature for multi-port-service , so it would be more appropriate, IMHO, to put it here, in general config loading.